docs/features/techdocs/using-cloud-storage.md

```diff
@@ -425,3 +422,61 @@ store and read the static generated documentation files. When you start the
 backend of the app, you should be able to see
 `techdocs info Successfully connected to the OpenStack Swift Storage container`
 in the logs.
+
+## Bonus: Migration from old OpenStack Swift Configuration
```

Comment:
This is a matter of taste, but you could move this part to the changeset instead. It will be added to the CHANGELOG.md after the next release and the library users can find it there.

Reply:
done that :)